core/web/testdata/oracle-spec.toml

```diff
@@ -5,7 +5,7 @@ p2pPeerID          = "12D3KooWCJUPKsYAnCRTQ7SUNULt4Z9qF8Uk1xadhCs7e9M711Lp"
 p2pBootstrapPeers  = [
     "/dns4/chain.link/tcp/1234/p2p/16Uiu2HAm58SP7UL8zsnpeuwHfytLocaqgnyaYKP8wu7qRdrixLju",
 ]
-isBootstrapPeer    = false
+isBootstrapPeer    = true
```

Comment:
Surprised this doesnt break any other tests...

Perhaps create a new fixture for bootstrappers?

Reply:
AFAICT this is currently used only in the OCR controller tests. Also, I changed it so that the default is a non-empty value. I don't think we need another file for the tests I'm running.